anaconda.py

```diff
@@ -576,6 +573,7 @@ def _earlyExceptionHandler(ty, value, traceback):
 
     from pyanaconda.kickstart import check_kickstart_error
     from pyanaconda.modules.common.constants.objects import SNAPSHOT
+    from pyanaconda.modules.common.constants.services import STORAGE
     from pyanaconda.modules.common.task import sync_run_task
```

Comment:
If I'm reading the code right, this is already imported on line 411: https://github.com/rhinstaller/anaconda/pull/6501/files#diff-251fc9d264c3bb59e93e192927bc04ccf8ba9fbde770b8b3623bb35b51885e7dR411

```suggestion
    from pyanaconda.modules.common.constants.objects import SNAPSHOT
    from pyanaconda.modules.common.task import sync_run_task
```